src/support.sch

```diff
@@ -16,10 +17,40 @@
         <rule context="*">
             <assert test="
                 name()='article'
+                or name()='kwd'
+                or name()='kwd-group'
+                or name()='title'
             " role="warn">
                 &lt;<name/>&gt; is ignored.
             </assert>
         </rule>
     </pattern>
 
+    <pattern id="kwd-group-kwd">
+        <rule context="kwd-group">
+            <assert test="kwd" role="warn">
+                &lt;<name/>&gt; without any &lt;kwd&gt; is ignored.
+            </assert>
+        </rule>
+    </pattern>
+
+    <pattern id="kwd-group">
+        <rule context="kwd-group[not(title)][not(@kwd-group-type)]">
+            <assert test="
+                title
+                or @kwd-group-type
+            " role="warn">
+                &lt;<name/>&gt; must have a @kwd-group-type or a &lt;title&gt;.
```

Comment:
As it stands, we actually require a `@kwd-group-type`. Probably makes sense to allow only a `<title>` since that's used for the heading if it exists?

Reply:
Opened https://github.com/libero/browser/pull/48.